trace_viewer/core/trace_model/world_memory_dump.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2015 The Chromium Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+-->
+
+<link rel="import" href="/core/trace_model/timed_event.html">
+
+<script>
+'use strict';
+
+/**
+ * @fileoverview Provides the WorldMemoryDump class.
+ */
+tv.exportTo('tv.c.trace_model', function() {
+  /**
+   * The WorldMemoryDump represents a simultaneous memory dump of all processes.
```

Comment:
I think we should keep it singular. A single <tt>GlobalMemoryDump</tt> contains a <tt>ProcessMemoryDump</tt> for each process at a *single* time point.